File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02577/p02577_num7_parsed.py
def is_multiple_of_9(N):
    digits_sum = sum(int(digit) for digit in str(N))
    if digits_sum % 9 == 0:
        return "Yes"
    else:
        return "No"

N = int(input())
print(is_multiple_of_9(N))

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02577/p02577_num7_parsed.cpp
#include <iostream>
#include <string>

int main() {
    std::string N;
    std::cin >> N;

    int sum = 0;
    for (char c : N) {
        sum += c - '0';
    }

    if (sum % 9 == 0) {
        std::cout << "Yes\n";
    } else {
        std::cout << "No\n";
    }

    return 0;
}